### PR TITLE
move some functions to pkg

### PR DESCRIPTION
**Pull request by mgufrone**

### CODE REVIEW

File: skel/config/config.go [hunk 1]
```diff
@@ -1,34 +1,23 @@
...
 import (
-	"github.com/openware/pkg/database"
 	"github.com/openware/pkg/mngapi"
+	sonicCfg "github.com/openware/pkg/sonic/config"
 )
 
 // Config is the application configuration structure
 type Config struct {
-	Database database.Config `yaml:"database"`
+	Database DatabaseConfig `yaml:"database"`
```
Comment (vshatravenko) [inline skel/config/config.go:10]: Could we still use database.Config here?

File: skel/config/database.go [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+	Pass   string `env:"DATABASE_PASS" env-description:"Database user password"`
+	Pool   int    `yaml:"pool" env:"DATABASE_POOL" env-description:"Database pool size"`
+}
+
+// Timestamps adding time at the end of models
+type Timestamps struct {
+	CreatedAt time.Time `yaml:"created_at"`
+	UpdatedAt time.Time `yaml:"updated_at"`
+}
+
```
Comment (vshatravenko) [inline skel/config/database.go:20]: Could we have this in openware/pkg?

File: skel/daemons/license.go [hunk 3]
```diff
@@ -4,6 +4,8 @@ import (
 	"encoding/base64"
 	"encoding/json"
 	"fmt"
+	sonic "github.com/openware/pkg/sonic/config"
+	"github.com/openware/sonic/skel/config"
```
Comment (vshatravenko) [inline skel/daemons/license.go:8]: Would be great to just have one import for config

File: skel/database/database.go [hunk 4]
```diff
@@ -0,0 +1,64 @@
+package database
```
Comment (vshatravenko) [inline skel/database/database.go:1]: It would be nice to have this in openware/pkg

**Review by vshatravenko — CHANGES_REQUESTED** [review]

File: skel/config/config.go (hunk 1 above)
Comment (mgufrone) [reply skel/config/config.go:10]: in the recent `pkg`, the `database` config is removed not sure what's the reason. or just the `database.Config` you want to put it in sonic pkg?

**Pull request closed without merging**